cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -151,6 +153,7 @@
     private final ZookeeperSetOperator zkSetOperations;
     private final KafkaSetOperator kafkaSetOperations;
     private final RouteOperator routeOperations;
+    private final ConfigMapOperator configMapOperator;
```

Comment:
There should be no need for this (the superclass declares `protected final ConfigMapOperator configMapOperations;`)